exporter/signalfxexporter/internal/translation/logdata_to_signalfxv2_test.go

```diff
@@ -133,7 +133,8 @@ func TestLogDataToSignalFxEvents(t *testing.T) {
 }
 
 func mapToEventProps(m map[string]interface{}) []*sfxpb.Property {
-	var out []*sfxpb.Property
+	out := make([]*sfxpb.Property, len(m))
```

Comment:
I'd suggest doing this and keeping other lines in this function as is.
```suggestion
	out := make([]*sfxpb.Property, 0, len(m))
```